### ISSUE TITLE
S8 Lost, undead being backdrop to second cutscene (hard)

### ISSUE THREAD

**Issue by Konrad22** [cooljeanius/The_Earths_Gut#41]
When you defeat the last undead, the 'death' event of said undead doesn't happen until the cutscene is over, making them a pretty weird background.

**Comment by cooljeanius** [comment]
So, I don't think I noticed this one actually happening in the replay you sent me? Reattaching it here for reference.
[TEG-Lost_replay_20230207-234611.gz](https://github.com/cooljeanius/The_Earths_Gut/files/10717512/TEG-Lost_replay_20230207-234611.gz)

**Comment by Konrad22** [comment]
It happens both in the replay I have saved, and in the one you attached just now.

**Comment by cooljeanius** [comment]
hm, I guess I'll have to give it another watch...

**Comment by cooljeanius** [comment]
Yeah I just rewatched the replay attached above, and it didn't seem to happen that time... I wonder what was different between the time where it didn't happen, and the time where it did?

**Comment by Konrad22** [comment]
Considering that it happens for me, but not for you, I guess you won't be able to verify it. And I have no idea why the same replay would behave differently for you and for me.
I'd say we need a bigger sample size to figure this out.
